scratch/paste-rename-poc/index.html

```
@@ -0,0 +1,163 @@
+					current editor line, because rename does not always update the
+					just-inserted link in time. Dedup lists the attachment folder and
+					adds <code>-N</code> (or a prefix) from the largest existing number.
+				</li>
+			</ol>
+			<p>
+				Windows paste-from-Explorer keeps the original filename, so the
+				<code>Pasted image </code> prefix never appears. That is why Handle
+				all attachments exists. QuickAdd hits the same kind of miss today.
+				paste-image-rename will not see a QuickAdd paste unless Handle all
+				attachments is on, and even then it will rename against the
+				<em>active</code> note, not the capture target.
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </code> ]

(tag-pair)